language-support/scala/bindings-akka/src/main/scala/com/digitalasset/ledger/client/binding/LedgerClientBinding.scala

```diff
@@ -57,6 +57,7 @@ class LedgerClientBinding(
       templateSelector: TemplateSelector,
       startOffset: LedgerOffset,
       endOffset: Option[LedgerOffset],
+      token: Option[String],
```

Comment:
```suggestion
      token: Option[String] = None,
```
Should this have a default value to avoid breaking downstream?

Reply:
This is a good idea, thanks!